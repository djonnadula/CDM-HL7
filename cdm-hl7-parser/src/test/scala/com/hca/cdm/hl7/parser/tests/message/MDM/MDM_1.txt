MSH|^~\&||HHAH|||20161002171105|HZE9211|MDM^T08|EPIC_HHAH_MDM_7F774_1E2FB7_0_FAC1|P|2.3||Epic_CWDV|||||||
EVN|T08|20161002171059||UPDATE_NOTE|HZE9211^MCMINN^THOMAS^^^^^^AH^^^^^HHAH
PID||Q6487^^^CWTTXIURN^CWTTXIURN|2000116569^^^EPIC^MRN|Y000145173^^^HHOA^HHOA~L001263882^^^SDMC^SDMC~Q6487^^^CWTTXIURN^CWTTXIURN|CHRISTENSEN^JIMMIE^L||19311218|M|CHRISTENSEN^JIMMIE^L^~CHRISTENSEN^JIMMIE^LEE^|W|PO BOX 10^^ELGIN^TX^78621^USA^P||(512)281-4439^P^H||ENGLISH|M|||460-48-6918|||NOT HISPANIC||||||||N
PD1|||AUSTIN HEART HHO^^90000001|||||||||||||
PV1||O|AHHHMO^^^HHAH^^^^^^^||||MCMT^MCMINN^THOMAS^^^^^^PROVID^^^^PROVID~HZE9211^MCMINN^THOMAS^^^^^^THREEFOUR^^^^THREEFOUR||||||||||||23756799||||||||||||||||||||||||||20160928120300|||||||

