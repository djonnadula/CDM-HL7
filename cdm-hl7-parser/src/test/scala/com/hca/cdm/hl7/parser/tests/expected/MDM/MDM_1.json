{
  "0000.COMN": {
    "sending_facility": "HHAH",
    "message_control_id": "EPIC_HHAH_MDM_123_51233_5_FAC1",
    "medical_record_num": "2000116569",
    "medical_record_urn": "Y000145173",
    "patient_account_num": ""
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_facility": "HHAH",
    "date_time_of_message": "20150807371105",
    "security": "ARB1234",
    "message_type": {
      "message_code": "MDM",
      "trigger_event": "T08"
    },
    "message_control_id": "EPIC_HHAH_MDM_123_51233_5_FAC1",
    "processing_id": "P",
    "version_id": "2.3",
    "continuation_pointer": "Epic_CWDV"
  },
  "0002.EVN": {
    "event_type_code": "T08",
    "recorded_date_time": "20150807371059",
    "event_reason_code": "UPDATE_NOTE",
    "operator_id": {
      "person_id": "ARB1234",
      "family_name": "DONALD",
      "given_name": "ARBY",
      "assigning_authority": "AH",
      "assigning_facility": "HHAH"
    }
  },
  "0003.PID": {
    "patient_id": {
      "id_num": "Q6487",
      "assigning_authority": "CWTTXIURN",
      "id_type_code": "CWTTXIURN"
    },
    "patient_id_list": {
      "id_num": "2000116569",
      "assigning_authority": "EPIC",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": [
      {
        "id_num": "Y000145173",
        "assigning_authority": "HHOA",
        "id_type_code": "HHOA"
      },
      {
        "id_num": "L001263882",
        "assigning_authority": "SDMC",
        "id_type_code": "SDMC"
      },
      {
        "id_num": "Q6487",
        "assigning_authority": "CWTTXIURN",
        "id_type_code": "CWTTXIURN"
      }
    ],
    "patient_name": {
      "family_name": "MARRKEN",
      "given_name": "LONNARD",
      "second_and_further_given_names_or_initials_thereof": "L"
    },
    "date_time_of_birth": "19311218",
    "administrative_sex": "M",
    "patient_alias": [
      {
        "family_name": "MARRKEN",
        "given_name": "LONNARD",
        "second_and_further_given_names_or_initials_thereof": "L"
      },
      {
        "family_name": "MARRKEN",
        "given_name": "LONNARD",
        "second_and_further_given_names_or_initials_thereof": "LEE"
      }
    ],
    "race": "W",
    "patient_address": {
      "street_address": "1 RD",
      "city": "BAYLOR",
      "state_or_province": "TX",
      "zip_or_postal_code": "78978",
      "country": "USA",
      "address_type": "P"
    },
    "phone_num_home": {
      "telephone_num": "(999)999-9999",
      "telecommunication_use_code": "P",
      "telecommunication_equipment_type": "H"
    },
    "primary_language": "ENGLISH",
    "marital_status": "M",
    "ssn_num_patient": "999-99-9999",
    "ethnic_group": "NOT HISPANIC",
    "patient_death_indicator": "N"
  },
  "0004.PD1": {
    "patient_primary_facility": {
      "organization_name": "AUSTIN HEART HHO",
      "id_num": "90000001"
    }
  },
  "0005.PV1": {
    "patient_class": "O",
    "assigned_patient_location": {
      "point_of_care": "AHHHMO",
      "facility": "HHAH"
    },
    "attending_doctor": [
      {
        "person_id": "MCMT",
        "family_name": "DONALD",
        "given_name": "ARBY",
        "assigning_authority": "PROVID",
        "id_type_code": "PROVID"
      },
      {
        "person_id": "ARB1234",
        "family_name": "DONALD",
        "given_name": "ARBY",
        "assigning_authority": "THREEFOUR",
        "id_type_code": "THREEFOUR"
      }
    ],
    "visit_num": "23756799",
    "discharge_date_time": "20160928120300"
  }
}